app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee

```diff
@@ -86,10 +87,17 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
   editDocument: (e) ->
     # Fire a global event. Consider refactoring this in the style of notifyExpandListeners above.
     Backbone.trigger 'LoanQuestionnairesView:edit', @
+    @flexibleTextArea()
 
   # Sets up the dirtyForm plugin on the questionnaire form.
   # We need to do this on a focus event because the form is not always visible when the page loads,
   # and dirtyForms doesn't seem to work with a hidden form. The focus event can only be fired when the
   # form is visible. Also, calling .dirtyForms() on each focus event doesn't seem to cause any issues.
   setupDirtyForm: (e) ->
     @$('.questionnaire-form form').dirtyForms()
+
+  flexibleTextArea: (e) ->
+    $('.answer-wrapper textarea')
+    .on 'change keyup keypress keydown paste cut clear', ->
+      $(this).height(0).height @scrollHeight
+    .change()
```

Comment:
@arisah-ijeoma No worries on this one. I'll check in with @hooverlunch about this. Thank you!